lightning/src/chain/keysinterface.rs

```diff
@@ -206,6 +233,10 @@ pub trait ChannelKeys : Send+Clone {
 	fn commitment_seed<'a>(&'a self) -> &'a [u8; 32];
 	/// Gets the local channel public keys and basepoints
 	fn pubkeys<'a>(&'a self) -> &'a ChannelPublicKeys;
+	/// Gets the key derivation parameters in case of a needed new derivation from
```

Comment:
Hmm, I think this could be much more clear. Suggested alternate wording:
```
+       /// Gets arbitrary identifiers describing this set of keys which are provided back to you in
+       /// some SpendableOutputDescriptor types. These should be sufficient to identify this
+       /// ChannelKeys object uniquely and lookup or re-derive its keys.
```